signal-desktop.sh

```diff
@@ -60,6 +84,19 @@ if [[ "${SIGNAL_PASSWORD_STORE}" == "basic" ]]; then
     fi
 fi
 
+# Explain filesystem=host upon the first run
+EXPLAIN_FILESYSTEM_HOST_FILE="${XDG_CONFIG_HOME}/Signal/explanation-filesystem-access-shown"
```

Comment:
Maybe just place these kind of files into `$XDG_CACHE_HOME` as they are no "official" config files of signal?

Reply:
Good idea. Could you PR this for this branch?